### PR TITLE
Remove star imports

### PR DESCRIPTION
**Pull request by ColCarroll**
Activating another pylint check.

### CODE REVIEW

File: pymc3/examples/GHME_2013.py [hunk 1]
```diff
@@ -1,20 +1,10 @@
-# -*- coding: utf-8 -*-
-# <nbformat>3.0</nbformat>
-
-# <codecell>
+import numpy as np
 import pandas as pd
-from pylab import *
+import pylab as pl
```
Comment (fonnesbeck) [inline pymc3/examples/GHME_2013.py]: I thought the convention is `import matplotlib.pyplot as plt`? I don't really care, other than for consistency.

Comment (twiecki) [reply pymc3/examples/GHME_2013.py]: Yes, that's a good point. I do care ;).

**Review by twiecki — APPROVED** [review]

**Comment by ColCarroll** [comment]
With the recent statistical test added, there's one job that takes ~30mins to run.  I can add a task to just add `pip install pymc3 && python -c 'from pymc3 import *'` to one of the other tasks, to make sure it has non-zero exit.

**Comment by twiecki** [comment]
> With the recent statistical test added, there's one job that takes ~30mins to run. I can add a task to just add pip install pymc3 && python -c 'from pymc3 import *' to one of the other tasks, to make sure it has non-zero exit.

That's a good idea.

**Pull request merged**